research/src/pages/select/select.proposal.mdx

```diff
@@ -116,11 +116,60 @@ children:
   Should this apply to the entire anatomy or solely the &lt;option&gt; and &lt;optgroup&gt; element
 </p>
 
+## Events <a class="link" href="#events" id="events"></a>
+
+### select
+
+| Event    | Behavior                                                                                         | Impacts          |
+| -------- | ------------------------------------------------------------------------------------------------ | ---------------- |
+| `change` | Updates the textContent property of the selected-value slot with the value property text content | textContent prop |
+
+### part button
+
+| Event          | Behavior                           | Impacts                                                                  |
+| -------------- | ---------------------------------- | ------------------------------------------------------------------------ |
+| `click`        | Toggles the open state on the host | [open](#open-state) state                                                |
```

Comment:
This may have already been discussed elsewhere, but should the open state not be toggled on `pointerdown`? I believe different OS's native counterparts handle this differently.

Reply:
Confirmed OSX v10.15.4 opens on `pointerdown`